projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -203,6 +206,8 @@ class_exists( 'Jetpack' ) ||
 
 		Assets::enqueue_script( 'jetpack-social-editor' );
 
+		wp_add_inline_script( 'jetpack-social-editor', $this->render_initial_state(), 'before' );
```

Comment:
Should this be added into (or replace) the `wp_localize_script` call below?

Reply:
We have shared state for both the editor and admin page, and then some variables unique to each. I can make a follow-up PR to improve how this works.

Reply:
Yeah it seems like we want all the initial state now for both the editor and the admin page. Perhaps we should just merge the arrays for now?

Reply:
Looking at this a bit more, I think longer term we should have some sort of `initial_state` class that can be used for rendering this data. I notice that there's some for the Connection package for example.